student.rb

```diff
@@ -2,6 +2,13 @@ class Student < Person
   def initialize(name, classroom, age = 'Unknown', parent_permission: true)
     super(name, age, parent_permission: parent_permission)
     @classroom = classroom
+    classroom.add_student(self)
```

Comment:
- [ ] You do need to pass the classroom object in your initialize method. Neither o you have to assign a Student object to a classroom when you create a Student object. The setter method classroom=(classroom) will do that for you. Kindly remove this from the initialize method. 👍

Reply:
The classroom was previously supposed to be created and assigned upon student creation. I will remove it from the parameters of student creation as well, so.

Reply:
This issue is well resolved! ✔️